portfolio/src/main/webapp/script.js

```diff
@@ -38,6 +39,17 @@ function deleteComments() {
   fetch('/delete-data', {method: 'post'}).then(getComments());
 }
 
+/**
+ * Validates whether text has been inputted before submitting.
+ */
+function validateText() {
+  const comment = document.getElementById('comment-box').value.trim();
+  if (comment == "") {
```

Comment:
I am curious here too unless @scottymb can give his insights on it. Should we use `trim()` here since `.value` may be null? But if not, how to prevent `\s\s` blank string with multiple spaces? Or for best practices, should we check all these cases including empty, null, blank?

Reply:
This is good - the `.value` of a textarea shouldn't be null, it would be an empty string if empty. I would actually recommend that you trim the value and check here for `if (!comment)`, i.e., checking the falsiness of the value. Trim will reduce all versions of only spaces and white space at the beginning and ending of a string, so you'll be left with either '' or a meaningful string. If it's falsey, you can be assured that nothing was entered, and that's all you need to know.